### PR TITLE
Bugfix/871/exeption page module

### PR DESCRIPTION
**Pull request by galoppi**

### CODE REVIEW

File: Classes/Provider/AbstractProvider.php [hunk 1]
```diff
@@ -274,7 +274,7 @@ public function getForm(array $row) {
 				$form = call_user_func_array(array($formClassName, 'create'), array($row));
 			} else {
 				$templatePathAndFilename = $this->getTemplatePathAndFilename($row);
-				if (NULL !== $templatePathAndFilename) {
+				if (NULL !== $templatePathAndFilename && '' != $templatePathAndFilename) {
```
Comment (jmverges) [inline Classes/Provider/AbstractProvider.php]: I think it should be TRUE === empty($templatePathAndFilename)

**Comment by xf-** [comment]
Please only push 1 issue in a PR.
-> 2 commits with different purposes.

**Comment by NamelessCoder** [comment]
The fix in e3aede76c3d34ef7f1d6ed346bb3eec34ec46c98 is already applied earlier; at the root of the problem rather than where template file gets used. Subject message of commit needs to be updated. Needs rebasing or cherry picking.

**Comment by NamelessCoder** [comment]
Both changes manually picked - thanks @galoppi!

**Pull request closed without merging**